### PR TITLE
Using /tmp

### PR DESCRIPTION
**Pull request by alexandre-mbm**
Do you accept this as solution for the issue #1?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+*~
```
Comment (albfan) [inline .gitignore]: remove this change. this is system related. I do not use editors that create that backup files. This should be in your `~/.git/info/exclude`

File: gitgraph [hunk 2]
```diff
@@ -12,6 +13,12 @@ QUIET=0
 
 ###############################################################################
 #
+
+TMP_HIGH="/tmp/$$-tmp.high"
+TMP_DOT="/tmp/$$-tmp.dot"
+TMP_SVG="/tmp/$$-out.svg"
+TMP_PNG="/tmp/$$-out.png"
+
```
Comment (albfan) [inline gitgraph]: All temporary files was removed in tip of master. Maybe useful to resolve #1. Let's think about it

File: gitgraph [hunk 3]
```diff
@@ -215,7 +215,7 @@ function create_dot_graph()
         # Highlight nodes.
         #
         if [[ "$HIGH" == "1" ]] &&
-           grep -q ^${shas[0]:0:5} "$TMP_HIGH"
+           grep -q ^${shas[0]:0:5} "$TMP_HIGH" 2>/dev/null
```
Comment (albfan) [inline gitgraph]: I will cherry pick this on master.

File: gitgraph [hunk 4]
```diff
@@ -69,11 +72,11 @@ function process_args()
             #
             CLEAN=1
             DEEP_CLEAN=1
-        elif [[ "$1" == "-p" ]]
+        elif [[ "$1" == "-d" ]]
         then
-            # Output as png (not svg)
```
Comment (albfan) [inline gitgraph]: Why? svg is a lossless format. svg is a better choose by default

File: gitgraph [hunk 5]
```diff
@@ -237,7 +237,7 @@ function create_dot_graph()
 
 function main()
 {
-    if ! check_dependencies dot dia eog git sed cut
```
Comment (albfan) [inline gitgraph]: This is already on tip of master

**Comment by albfan** [comment]
Thanks for that. I have commented what could be useful, but PR as a whole wil not be accepted.

File: gitgraph [hunk 6]
```diff
@@ -252,6 +255,8 @@ function main()
         then
             dia "$TMP_SVG"
             rm "$TMP_SVG"
+        else
```
Comment (albfan) [inline gitgraph]: Something to think about. Just do this if user request with an output file. Say:

- Add '--output' option.
- If output option is present pipe command with > to that file

relates with #1

File: gitgraph (hunk 4 above)
Comment (alexandre-mbm) [inline gitgraph]: PNG is easier to show; compatibility for Nautilus, browsers, email...

File: gitgraph (hunk 6 above)
Comment (alexandre-mbm) [inline gitgraph]: I have no time for more work. You can continue my branch. I would appreciate if my commits were to enter for the Git history in some way.

File: gitgraph (hunk 5 above)
Comment (alexandre-mbm) [inline gitgraph]: I'm in a hurry to analyzing code but all is looking like you have ignored my commits like if were your changes. :-1: 

Hint: cherry pick, or merge to change it after.

File: gitgraph (hunk 3 above)
Comment (alexandre-mbm) [inline gitgraph]: With your signature!  :-1:

Comment (alexandre-mbm) [inline gitgraph]: You could merge to rebase after...

File: .gitignore (hunk 1 above)
Comment (alexandre-mbm) [inline .gitignore]: It is your project. You can accept pull requests by doing some minor reversions after.

Comment (alexandre-mbm) [inline .gitignore]: If you want I can to do that for you, and teach it to you.

File: gitgraph (hunk 2 above)
Comment (alexandre-mbm) [inline gitgraph]: I would want to see my commits in the Git's history, even if edited after. Or parts (commits subsets) ofpull requests.

**Pull request closed without merging**